### PR TITLE
Improve Readme

### PR DESCRIPTION
**Pull request by Andre601**
- Added Gradle installation example
- Added missing `color(String)` and `font(String)` methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +24,23 @@ For those of you using Maven, just add the following to your `pom.xml` file:
...
+
+```gradle
+repositories {
+    maven { url = "https://rayzr.dev/repo/" }
+}
+```
+
+```gradle
+dependencies {
+    compile('me.rayzr522:jsonmessage:1.2.0')
```
Comment (Rayzr522) [inline README.md:38]:
```suggestion
    compile 'me.rayzr522:jsonmessage:1.2.0'
```

File: README.md [hunk 2]
```diff
@@ -23,6 +24,23 @@ For those of you using Maven, just add the following to your `pom.xml` file:
 </dependency>
 ```
 
+### Gradle
+If you're using Gradle, add this to your `build.gradle` file:
+
+```gradle
+repositories {
+    maven { url = "https://rayzr.dev/repo/" }
```
Comment (Rayzr522) [inline README.md:32]:
```suggestion
    maven { url 'https://rayzr.dev/repo/' }
```

File: README.md [hunk 3]
```diff
@@ -151,6 +169,8 @@ Method             | Description
 `bar(int)`         | Creates a horizontal divider bar of the given length
 `create(String)`   | Creates a new JSONMessage with the given text as a starting point
 `color(ChatColor)` | Sets the color of the current message part
+`color(String)`    | Same as `color(ChatColor)` but allows the usage of HEX colors in 1.16 and newer
```
Comment (Rayzr522) [inline README.md:172]: hmmmm. should we mention here that they can pass in a default ChatColor to use if it's not on 1.16? i feel like at this point actual javadocs should be used to generate a docs site...

Comment (Andre601) [reply README.md:172]: Seems like an idea.

Comment (Rayzr522) [reply README.md:172]: hmm. i think it's fine how it is for now, there are javadocs in the code which can be used anyways if people really want, and I want to keep these docs simple. eventually I'll move to actual published javadocs at some point :crossed_fingers:

**Comment by Rayzr522** [comment]
@Andre601 if you don't mind, i'm gonna merge this really quick so I can make final changes on master with the reflection optimizations and release 1.2.1 (to address #8)

**Comment by Andre601** [comment]
Sure, go ahead.

**Pull request merged**